carbonmark/components/pages/Project/Retire/Pool/RetireForm.tsx

```diff
@@ -93,27 +94,26 @@ export const RetireForm: FC<Props> = (props) => {
   useEffect(() => {
     const getFiatMaxBalance = async () => {
       try {
-        const fiatInfo = await getFiatInfo();
+        const { MAX_USDC } = await getFiatInfo();
```

Comment:
Let's just remove this async logic and keep use a constant `MAX_USDC` value of `2000`. Offsetra agreed to keep this much liquidity at all times.

The staging wallet only has a few bux in it, but that's OK because when the user does a $2000 checkout in Staging via Stripe Test Mode, it will default to `0.002 tonnes` under the hood